src/utils/has-flags.ts

```diff
@@ -0,0 +1,19 @@
+import type * as d from '../declarations';
+
+/**
+ * Does the command have the debug flag?
+ * @param config The config passed into the Stencil command
+ * @returns true if --debug has been passed, otherwise false
+ */
+export function hasDebug(config: d.Config) {
+  return config.flags.debug;
```

Comment:
IIRC, we were making some assumptions around `flags` being truthy in the original implementation.  If we move this helper to be generally used by others, I don't think that assumption holds.  We should protect ourselves with a small tweak:
```suggestion
  return config.flags?.debug;
```

Can we add a test for this case?